iele-examples/erc20.iele

```diff
@@ -1,21 +1,30 @@
+// Copyright (c) 2017 Runtime Verification, Inc. All Rights Reserved.
+//
+// This contract implements the ERC-20 token standard.
 contract ERC20 {
-  @addressShift = 1461501637330902918203684832716283019655932542976 // 2^160
 
-// storage layout
-  // variable location
+  // constant equal to 2^160, used for hashing
```

Comment:
Hashing isn't really the right word for how we are constructing the storage keys...

Reply:
I just removed the ",used for hashing" part.